pkg/services/object/put/distributed.go

```diff
@@ -115,6 +115,10 @@ func (x errIncompletePut) Error() string {
 	return commonMsg
 }
 
+func (x errIncompletePut) Unwrap() error {
```

Comment:
Can you give an example of what message is expected now? Is it still obvious that there was some error on _others_ node.

Reply:
With that PR error messages would be just the deepest error because the current error messaging is [designed](https://github.com/nspcc-dev/neofs-node/blob/42554a9298a32cff2669039e21c44293587200ce/pkg/services/util/sign.go#L224-L227) to be so

I agree that keeping the chain could be a good error context and that is why im not sure about the PR. The problem that PR solves is described in the linked issue: any SDK error that has not been converted to its V2 form [does not produce](https://github.com/nspcc-dev/neofs-sdk-go/blob/b4b07a3c4e11a2f5123cdd49f30fa42d43d9dee1/client/status/object.go#L34) a text message (wrapping an SDK error by an error that does not have `Unwrap` method leads to calling `Error` method without conversion to a `V2` struct => no error message). I did not like such a solution but here we are.

If our desire to save error chain _in an "incomplete" context_ is really strong, I may suggest using `init()` in SDK for predefined errors that fills all error messages before their usage. Another solution could be introducing a new "IncompleteObjectPut" API error that wraps any error and has a detailed description of error numbers and reasons

Reply:
For now I don't mind